lib/credit_card_sanitizer.rb

```diff
@@ -59,7 +60,7 @@ def initialize(options = {})
   # Returns a String of the redacted text if a credit card number was detected.
   # Returns nil if no credit card numbers were detected.
   def sanitize!(text)
-    to_utf8!(text)
+    text.scrub!('�')
```

Comment:
Is it necessary to explicitly specify the replacement char? It looks like [scrub_rb  defaults to it](https://github.com/jrochkind/scrub_rb/blob/master/lib/scrub_rb.rb#L15-L23)